app/controllers/credentials.controller.js

```diff
@@ -118,23 +118,25 @@ module.exports = {
     var url = connectorUrl.replace('{accountId}', accountId)
     var correlationId = req.headers[CORRELATION_HEADER] || ''
 
-    connectorClient().postAccountNotificationCredentials({
-      payload: { username, password },
-      correlationId: correlationId,
-      gatewayAccountId: accountId
-    }, function (connectorData, connectorResponse) {
-      var duration = new Date() - startTime
+    try {
+      await connectorClient().postAccountNotificationCredentials({
+        payload: { username, password },
+        correlationId: correlationId,
+        gatewayAccountId: accountId
+      })
+
+      let duration = new Date() - startTime
```

Comment:
this can be `const`

Reply:
Updated above and the rest in this PR https://github.com/alphagov/pay-selfservice/pull/2445